Better test coverage, use same template for running tests

jtveiten opened the pull request:
I've updated the tests to adhere more to the approach in Base. Also added a bit more coverage with tests along the y axis.

jtveiten committed:
Better test coverage, use same template for running tests

jtveiten committed:
Forgot MIT header in test file

ahojukka5 commented:
https://coveralls.io/builds/15313309/source?filename=src%2FFEMTruss.jl

You get quite easily coverage to 100 % just by calling the functions shown in red in report.

jtveiten committed:
Add one more function test

jtveiten commented:
I got into a problem when calling get_formulation_type(p1)
It comes up with an undefined variable when called. Is there an issue in Base, or am I doing something wrong?
Thus I just checked in the first function check

jtveiten committed:
Add test file for truss systems

jtveiten committed:
Forgot license header again

ahojukka5 committed:
Fix test

`get_formulation_type` is not exported from FEMBase

ahojukka5 committed:
Implement nodal boundary conditions

* Using element `Poi1`, one can use field `nodal force i` where `i` is a
  dof number, to set nodal force / concentrated load (CLOAD in Abaqus).
* Using element `Poi1`, one can use field `fixed displacement i`, where
  `i` is a dof number, to fix displacement in some node.
* Wrote a test to test nodal elements + 2 truss test.

jtveiten committed:
Merge pull request #1 from ahojukka5/master

Implement nodal boundary conditions

jtveiten committed:
Added checks for test, missing element and support force checks

ahojukka5 commented:
Looks good to me, maybe we now merge this now and then plan if some changes is needed.
